[2023-06-15T11:31:02.168+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry_dag.add_entry_task scheduled__2023-06-15T11:30:00+00:00 [queued]>
[2023-06-15T11:31:02.175+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry_dag.add_entry_task scheduled__2023-06-15T11:30:00+00:00 [queued]>
[2023-06-15T11:31:02.175+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T11:31:02.176+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-06-15T11:31:02.176+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T11:31:02.183+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): add_entry_task> on 2023-06-15 11:30:00+00:00
[2023-06-15T11:31:02.188+0000] {standard_task_runner.py:55} INFO - Started process 4792 to run task
[2023-06-15T11:31:02.191+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'add_entry_dag', 'add_entry_task', 'scheduled__2023-06-15T11:30:00+00:00', '--job-id', '149', '--raw', '--subdir', 'DAGS_FOLDER/postgres_dag.py', '--cfg-path', '/tmp/tmpp_sss3ub']
[2023-06-15T11:31:02.192+0000] {standard_task_runner.py:83} INFO - Job 149: Subtask add_entry_task
[2023-06-15T11:31:02.243+0000] {task_command.py:389} INFO - Running <TaskInstance: add_entry_dag.add_entry_task scheduled__2023-06-15T11:30:00+00:00 [running]> on host ff7147a0d5a7
[2023-06-15T11:31:02.291+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=add_entry_dag
AIRFLOW_CTX_TASK_ID=add_entry_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-15T11:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-15T11:30:00+00:00
[2023-06-15T11:31:02.300+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-06-15T11:31:02.305+0000] {sql.py:364} INFO - Running statement: INSERT INTO assignment_table (execution_time) VALUES ('2023-06-15 11:31:02.292694'), parameters: None
[2023-06-15T11:31:02.306+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-15T11:31:02.307+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-15T11:31:02.316+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=add_entry_dag, task_id=add_entry_task, execution_date=20230615T113000, start_date=20230615T113102, end_date=20230615T113102
[2023-06-15T11:31:02.364+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-15T11:31:02.377+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
